ydb/core/driver_lib/cli_utils/cli_cmds_tenant.cpp

```diff
@@ -606,6 +620,40 @@ class TClientCommandTenantAddPools
     }
 };
 
+class TClientCommandTenantChangeQuotas
+    : public TTenantClientGRpcCommand<Ydb::Cms::V1::CmsService,
+                                      Ydb::Cms::AlterDatabaseRequest,
+                                      Ydb::Cms::AlterDatabaseResponse,
+                                      decltype(&Ydb::Cms::V1::CmsService::Stub::AsyncAlterDatabase),
+                                      &Ydb::Cms::V1::CmsService::Stub::AsyncAlterDatabase> {
+
+    uint64_t DataSizeHardQuota;
+    uint64_t DataSizeSoftQuota;
+public:
+    TClientCommandTenantChangeQuotas()
+        : TTenantClientGRpcCommand("change", {}, "Change data size quotas for database")
+    {}
+
+    void Config(TConfig& config) override {
+        TTenantClientGRpcCommand::Config(config);
+        config.SetFreeArgsMin(0);
+        config.Opts->AddLongOption("data-size-hard-quota", "A maximum data size in bytes, new data will be rejected when exceeded")
+            .RequiredArgument("NUM").StoreResult(&DataSizeHardQuota);
```

Comment:
All options are optional by default. If you want to make it mandatory, use the Required() method.